pkg/tests/conftest.py

```diff
@@ -26,12 +26,12 @@
 log = logging.getLogger(__name__)
 
 
-@pytest.fixture(scope="session")
+@pytest.fixture
```

Comment:
```suggestion
@pytest.fixture
```
We still want the reduced fixture scope?